src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -660,9 +663,19 @@ namespace OpenMS
       if (cexp_->getInstrument() != Instrument() || cexp_->getContacts().size() != 0)
       {
         const Instrument& inst = cexp_->getInstrument();
+        // the Instrument Manufacturer is paramount for some downstream tools
+        // Since the .getVendor() is usually empty, we infer this via the Aquisiton Software, which is unique to Thermo
+        String manufacturer = inst.getVendor();
+        if (String(inst.getSoftware().getName()).toLower().hasSubstring("xcalibur"))
```

Comment:
I'm a bit hesitant here, since we'd basically want exactly this manufacturer. Even it already says 'Thermo', it should say 'Thermo Scientific'.
But there is other things which must fit to be MQ compatible, so I might just introduce a 'forceMQcompatibility' flag.